Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -2224,17 +2224,17 @@ void WebProcess::setAppBadge(std::optional<WebPageProxyIdentifier> pageIdentifie
 #if ENABLE(WEB_PUSH_NOTIFICATIONS)
     if (DeprecatedGlobalSettings::builtInNotificationsEnabled()) {
         if (m_sessionID)
-            ensureNetworkProcessConnection().connection().send(Messages::NotificationManagerMessageHandler::SetAppBadge({ origin, badge }), m_sessionID->toUInt64());
+            ensureProtectedNetworkProcessConnection()->protectedConnection()->send(Messages::NotificationManagerMessageHandler::SetAppBadge({ origin, badge }), m_sessionID->toUInt64());
```

Comment:
Same comment as above. I suspect we don't need ensureProtectedNetworkProcessConnection().